02.API-REFERENCE.md

```diff
@@ -660,6 +729,8 @@ Possible values:
  - JERRY_TYPEDARRAY_INT32 - represents the Int32Array TypedArray
  - JERRY_TYPEDARRAY_FLOAT32 - represents the Float32Array TypedArray
  - JERRY_TYPEDARRAY_FLOAT64 - represents the Float64Array TypedArray
+ - JERRY_TYPEDARRAY_BIGINT64 - represents the BigInt64Array TypedArray
+ - JERRY_TYPEDARRAY_BIGUINT64 - represents the BigUint64Array TypedArray
```

Comment:
Please add *Changed in version [[NEXT_RELEASE]]*: Added `JERRY_TYPEDARRAY_BIGINT64`, `JERRY_TYPEDARRAY_BIGUINT64` values.

Reply:
Ok, I am going to add it to the doc first, then update this PR.

Reply:
I've added the suggested lines as https://github.com/jerryscript-project/jerryscript/pull/4259 landed.